tests/e2e/upgrade-vertica/35-update-auto-restart.yaml

```diff
@@ -0,0 +1,20 @@
+# (c) Copyright [2021] Micro Focus or one of its affiliates.
```

Comment:
What is the purpose of step 35 and beyond?  We already tested that we can call upgrade-vertica.sh when all pods aren't running.  We have a cluster that fully up at this point, so we are just going from vertica-k8s:latest to vertica-k8s:10.1.1-0?

Reply:
The pods are running but all the nodes are down. I set autoRestartVertica to false and then deleted the sts. New pods are automatically created but all the vertica nodes will be down. This was to emulate the case we have all pods are running but the cluster is down.

Reply:
Okay, lets add a comment at the start of this test to explain the purpose of the next steps.